pkg/api/grpcServer.go

```diff
@@ -143,7 +143,9 @@ func (s *GRPCServer) Stop() {
 type handlerFunc func(tracker attribute.Tracker, request proto.Message, response proto.Message)
 
 func (s *GRPCServer) streamLoop(stream grpc.ServerStream, request proto.Message, response proto.Message, handler handlerFunc) error {
-	tracker := s.attrMgr.NewTracker()
+	tracker := s.attrMgr.GetTracker()
```

Comment:
The usual Go style is to avoid the word "Get" if it doesn't add value. I'd just call the function "Tracker".